jujugui/static/gui/src/app/components/entity-details/entity-details.js

```diff
@@ -92,6 +92,37 @@ YUI.add('entity-details', function() {
       });
     },
 
+    /**
+      Get the charm icon or display the default bundle icon.
+
+      @method _getIcon
+      @returns {String} The icon URL
+    */
+    _getIcon: function(entity) {
+      var icon;
+      if (entity.type === 'bundle') {
+        icon = '/juju-ui/assets/images/non-sprites/bundle.svg';
+      } else {
+        icon = entity.iconPath;
+      }
+      return icon;
+    },
+
+    /**
+      Get the element for the series.
+
+      @method _getSeries
+      @returns {String} The icon URL
+    */
+    _getSeries: function(series) {
+      console.log(series);
+      if (series) {
+        return <li className="header__series">{series}</li>;
+      } else {
+        return null;
```

Comment:
Does null have any special functionality vs `undefined` as if you didn't explicitly return?

You can drop the `else` though